src/http/web_socket.cr

```diff
@@ -73,6 +76,12 @@ class HTTP::WebSocket
           @on_message.try &.call(@current_message.to_s)
           @current_message.clear
         end
+      when Protocol::Opcode::BINARY
+        @current_message.write @buffer[0, info.size]
```

Comment:
I know nothing about the websocket protocol, so excuse me in case it's obvious. So binary message are too at maximum 4096 bytes big?